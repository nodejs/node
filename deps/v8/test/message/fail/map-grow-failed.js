// Copyright 2021 the V8 project authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

const kMapSizeLimit = 1 << 24;
let m = new Map();
for (let i = 0; i < kMapSizeLimit + 1; i++) {
  m.set(i, 0);
}
