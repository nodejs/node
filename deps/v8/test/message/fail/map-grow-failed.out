*%(basename)s:{NUMBER}: RangeError: Map maximum size exceeded
  m.set(i, 0);
    ^
RangeError: Map maximum size exceeded
    at Map.set (<anonymous>)
    at *%(basename)s:{NUMBER}:{NUMBER}
